packages/http-client-python/eng/scripts/ci/regenerate.ts

```diff
@@ -57,59 +57,84 @@ const EMITTER_OPTIONS: Record<string, Record<string, string> | Record<string, st
   },
   "type/array": {
     "package-name": "typetest-array",
+    "enable-typespec-namespace": "false",
```

Comment:
When enable typespec namespace, CI in windows will report error:
![image](https://github.com/user-attachments/assets/9b67d741-61f8-43c0-8906-4411ba8c573b)

I can't figure it out for now so disable typespec namespace for cases under `type/` folder.